### ISSUE TITLE
SRB thrust curve fix

### ISSUE THREAD

**Issue by raidernick** [KSP-RO/RealismOverhaul#2001]
@pap1723 I'm leaving this here as a note for the castor srb thrust curves.

The castor 1:
-Only reaches 80% of it's rated thrust, needs to be fixed

The castor 2:
-The current thrust curve explodes, and if you edit the max temp so that it doesn't, it never burns out. It has residuals starting at 150 units of fuel and keep burning for at least 20 minutes. Needs to be completely redone.

**Comment by MikeOnTea** [comment]
The castor 1 curve should be fine, it reaches 100 % thrust only for a part of a second after ignition, see the dashed curve here: 
![image](https://user-images.githubusercontent.com/11480934/53911887-ffd51d80-4057-11e9-9c77-0e77c15ea495.png)
We'll fix the curve for the castor 2.

**Comment by raidernick** [comment]
the castor one curve does not work with the 2, I tried just swapping it, it adds huge residuals to it and the srb never burns out and only burns 70% of it's fuel.

**Comment by MikeOnTea** [comment]
Yes, that's probably because you didn't adjust thrust? Thrust needs to be adjusted for the curve, because our current thrust values most probably aren't peak thrust but rather the thrust of the plateau you see in the curve above. The curve thus needs a thrust increase by ~65/51, but that might still be not enough to make it burn all of its fuel, which either suggests the real castor 2 had a different curve (with a longer plateau/less long tail) or some of our other values are off, either not enough thrust or too much fuel?

**Comment by pap1723** [comment]
I have the real Castor 2 thrust curve. I will make the changes tonight.

**Comment by raidernick** [comment]
I have no idea engines are not my area of expertise. I already discussed this issue at length with pap nathankell starwaster and blowfishpro on the discord yesterday and I'm assuming pap made a decision on how to move forward with fixing it as something is causing the engine internal temperatures to be higher in 1.4.5 then they were previously in 1.3.1. I only made this issue as a reminder and to keep track of the problem. You'll need to discuss it with them but I advise you do nothing to fix it because they already have a plan to do it.
